clientv3/health_balancer.go

```diff
@@ -48,8 +48,8 @@ type healthBalancer struct {
 	// eps stores all client endpoints
 	eps []string
 
-	// unhealthy tracks the last unhealthy time of endpoints.
-	unhealthy map[string]time.Time
+	// unhealthyHosts tracks unhealthy hosts.
+	unhealthyHosts map[string]time.Time
```

Comment:
Was trying to differentiate from `up(addr grpc.Address)` where `addr` is `[scheme]://[host]:[port]`, where `hb.host2ep` contains `[host]:[port]`. `addr` is kind of confusing because we actually pass keys from `host2ep`. What do you think?

Reply:
addr == endpoint, which contains the fqdn (scheme://host:port). we need to make this accurate. if it is host, then use host. or use endpoint or addr.

Reply:
`endpoint` sounds better. Will change to `endpoint`.

Reply:
if you want to be accurate, then change everything to hostport if it only contains host:port.